Source/SwiftLintFramework/Rules/FirebaseConfigFetchRule.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  FirebaseConfigFetchRule.swift
+//  SwiftLint
+//
+//  Created by Ibrahim Ulukaya (Google Inc.) on 3/8/17.
+//  Copyright © 2017 Realm. All rights reserved.
+//
+
+import SourceKittenFramework
+
+public struct FirebaseConfigFetchRule: ConfigurationProviderRule, OptInRule {
+
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "firebase_config_fetch",
+        name: "Firebase Config Fetch",
+        description: "Firebase Config fetch should be called.",
+        nonTriggeringExamples: [
+            "class ViewController: UIViewController {\n" +
+            "  override func viewDidLoad() {\n" +
+            "    super.viewDidLoad() \n" +
+            "    remoteConfig.fetch(withExpirationDuration: TimeInterval(expirationDuration)) {" +
+            "        (status, error) -> Void in \n" +
+            "    }\n" +
+            "  }\n" +
+            "}"
+        ],
+        triggeringExamples: [
+            "class ViewController: UIViewController {\n" +
```

Comment:
Now instead of it's availability, it checks that if it's called, it should be called inside viewdidload